.circleci/config.yml

```diff
@@ -0,0 +1,218 @@
+      - run:
+          name: install-npm
+          command: npm ci
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+      - run:
+          name: Install test dependencies
+          command: |
+            GECKODRIVER_URL=$(
+              curl -s 'https://api.github.com/repos/mozilla/geckodriver/releases/latest' |
+              python -c "import sys, json; r = json.load(sys.stdin); print([a for a in r['assets'] if 'linux64' in a['name']][0]['browser_download_url']);"
+            );
+            curl -L -o geckodriver.tar.gz $GECKODRIVER_URL
+            gunzip -c geckodriver.tar.gz | tar xopf -
+            chmod +x geckodriver
+            sudo mv geckodriver /bin
+            geckodriver --version
+      - run:
+          name: Install Firefox
+          command: |
+            sudo apt-get install libgtk-3-dev libdbus-glib-1-2
+            sudo npm i -g get-firefox
+            get-firefox --branch nightly --platform linux --extract --target ~/
+            echo 'export PATH="$HOME/firefox:$PATH"' >> $BASH_ENV
+      - run:
+          name: Verify Firefox
+          command: |
+            firefox --version
+      - run:
+          name: test
+          command: npm test
+      - run:
+          name: coverage
+          command: npm run codecov
+      - run:
+          name: integration test
+          command: npm run integration
+          environment:
+            LOCKBOX_FIREFOX_HEADLESS: "1"
+  package-addon-staging:
+    docker:
+      - image: circleci/node
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@6.7'
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: install-npm
+          command: npm ci
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+      - run:
+          name: Package addon for staging
+          command: |
+            npm run package
+            npm run sign:stage
+      - store_artifacts:
+          path: /tmp/signed-stage.xpi
+          destination: signed-stage.xpi
+
+  package-addon-production:
+    docker:
+      - image: circleci/node
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@6.7'
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: install-npm
+          command: npm ci
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+      - run:
+          name: Package addon for production
+          command: |
+            npm run package
+            npm run sign:prod
+      - store_artifacts:
+          path: ~/project/addons/signed-prod.xpi
```

Comment:
maybe add a `destination` for a simplified `signed-addon.xpi` or `lockbox-signed.xpi`?